ml/xgboost/src/test/java/ai/djl/ml/xgboost/XgbModelTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package ai.djl.ml.xgboost;
+
+import ai.djl.MalformedModelException;
+import ai.djl.Model;
+import ai.djl.inference.Predictor;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.Shape;
+import ai.djl.training.util.DownloadUtils;
+import ai.djl.translate.NoopTranslator;
+import ai.djl.translate.TranslateException;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class XgbModelTest {
+
+    @BeforeClass
+    public void downloadXGBoostModel() throws IOException {
+        Path modelDir = Paths.get("build/model");
+        DownloadUtils.download(
+                "https://resources.djl.ai/test-models/xgboost/regression.json",
```

Comment:
how big is the json file? better check in to test/resources folder.

Reply:
Around 114kb